introduction.md

```diff
@@ -0,0 +1 @@
+I am Peter
```

Comment:
I am done.

2015-09-05 10:54 GMT+08:00 Code School Kiddo <notifications@github.com>:

> In introduction.md
> <https://github.com/deadlyvipers/dojo_rules/pull/6633#discussion_r38806545>
> :
>
> > @@ -0,0 +1 @@
> > +I am Peter
>
> Looks good! Could you also please mention your favorite Code School path
> in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or
> Electives.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/deadlyvipers/dojo_rules/pull/6633/files#r38806545>.
>